spec/factories/legal_aid_applications.rb

```diff
@@ -47,6 +47,14 @@
     end
 
     trait :with_applicant_and_partner do
+      transient do
+        with_bank_accounts { 0 }
+      end
+      applicant { build(:applicant, :with_address, :with_partner, with_bank_accounts:) }
+      partner { build(:partner) }
```

Comment:
Do you need `partner { build(:partner) }` I would have expected `:with_partner` to have created one

Reply:
Looks like `:with_partner`  just adds `has_partner { true }` to the applicant